app/layout.tsx

```diff
@@ -33,8 +35,10 @@ export const metadata: Metadata = {
 export default function RootLayout({ children }: { children: ReactNode }) {
   return (
     <html lang="en">
-      <Toaster />
-      <body className={cx(cal.variable, inter.variable)}>{children}</body>
+      <body className={cx(cal.variable, inter.variable)}>
+        {children}
+        <Toaster />
```

Comment:
I moved the toaster because it renders a `section` so it can't be a child of `html`.


<img width="455" alt="image" src="https://github.com/steven-tey/novel/assets/15332326/471690f9-29d6-47c5-b98c-265fe3f69ee2">

Reply:
Perfect – this has actually been bugging me for a while haha so thank you!